examples/release_highlights/plot_release_highlights_0_22_0.py

```diff
@@ -12,7 +12,7 @@
 
 To install the latest version (with pip)::
 
-    pip install -U scikit-learn --upgrade
+    pip install -U scikit-learn
```

Comment:
why?

Reply:
I'm confused by this. As I understand, `-U` is the same as `--upgrade`, isn't it?

Reply:
lol I thought it was for `--user` (which I've never ever used)

but you're right, let's keep `--upgrade` and remove `-U` then